brain/converter/crypto.py

```diff
@@ -0,0 +1,44 @@
+'''@crypto
+
+This file contains various cryptography wrappers.
+
+
+'''
+
+
+import os
+import hashlib
+import base64
+
+saltlength = 32
+
+def hashpass(p):
+    '''@hashpass
+
+    This method returns a hash and salt from a password p
+    
+    @salt - a random string of saltlength bytes generated to hash the password
+    
+    '''
+    salt = base64.b64encode(os.urandom(saltlength))
+    return hashlib.sha512(salt + p).hexdigest()+"$"+salt
+
+def verifypass(p, h):
+    '''@verifypass
+
+    This function verifies that a password p hashes to a hash h as 
+    returned by hashpass.
+
+    @h - hash extracted from the hash+salt
+    @s - salt extracted from the hash+salt
+
+    '''
+    h,s = h.split('$')
+    return hashlib.sha512(s + p).hexdigest() == h
+
+#hash1 = hashpass('password')
+#hash2 = hashpass('password')
+
+#print 'hashpass returns different values for same password:', hash1 != hash2
```

Comment:
This line is breaking the linter since the line is greater than 79 characters.  You could probably shorten `'hashpass returns different values for same password:'`, or just remove that line entirely.